numerics/log_b.hpp

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+#include <limits>
+
+namespace principia {
+namespace numerics {
+
+// A constexpr implementation of the IEEE 754:2008 logB function.
+// Uses sourceFormat as logBFormat, which makes it easy to cleanly handle NaN,
```

Comment:
Weird camelCase (2x).

Reply:
IEEE 754 uses that case, and I am referring to names in there.